pkg/test/resourcefixture/sets.go

```diff
@@ -108,6 +108,7 @@ func IsPureDirectResource(gk schema.GroupKind) bool {
 		"SecureSourceManagerInstance",
 		"SecureSourceManagerRepository",
 		"ManagedKafkaCluster",
+		"ManagedKafkaTopic",
```

Comment:
@maqiuyujoyce Could you confirm if we need this hack after you removed direct from the service mapping path?